Formula/xcftools.rb

```diff
@@ -0,0 +1,52 @@
+class Xcftools < Formula
+  desc "Tools for GIMP's native file format XCF"
+  homepage "http://henning.makholm.net/software"
+  url "http://henning.makholm.net/xcftools/xcftools-1.0.7.tar.gz"
+  sha256 "1ebf6d8405348600bc551712d9e4f7c33cc83e416804709f68d0700afde920a6"
+
+  depends_on "libpng"
+
+  def install
+    # Avoid deprecated use of libpng 1.2
+    inreplace "xcf2png.c", /png_(voidp|error_ptr)_NULL/, "NULL"
```

Comment:
The developer's website does not have patch submission page. Probably need to send an email the author instead. Meanwhile, I left a link to the [debian patch](http://anonscm.debian.org/cgit/collab-maint/xcftools.git/commit/?id=c40088b82c6a788792aae4068ddc8458de313a9b).